bgpd/bgpd.h

```diff
@@ -1047,6 +1049,8 @@ struct peer {
 	struct stream_fifo *ibuf; // packets waiting to be processed
 	struct stream_fifo *obuf; // packets waiting to be written
 
+	/* used as a block to deposit raw wire data to */
+	uint8_t ibuf_scratch[BGP_MAX_PACKET_SIZE * BGP_READ_PACKET_MAX];
```

Comment:
I'm just double-checking: is this used somewhere? Or I'm blind.

Reply:
I see it's part of https://github.com/FRRouting/frr/pull/8570

Reply:
ech, yeah that should be deleted here

Reply:
definitely, it will take a lot of memory; it would be nice to aggregate all ibuf_scratch traffic in a common place for all peers.